src/components/LayoutTree.tsx

```diff
@@ -471,87 +631,59 @@ const TreeNode = ({
-        })}
-        style={{
-          // Cannot use transform because it creates a new stacking context, which causes later siblings' DropChild to be covered by previous siblings'.
-          // Unfortunately left causes layout recalculation, so we may want to hoist DropChild into a parent and manually control the position.
-          left: x,
-          top: y,
-          // Table col1 uses its exact width since cannot extend to the right edge of the screen.
-          // All other thoughts extend to the right edge of the screen. We cannot use width auto as it causes the text to wrap continuously during the counter-indentation animation, which is jarring. Instead, use a fixed width of the available space so that it changes in a stepped fashion as depth changes and the word wrap will not be animated. Use x instead of depth in order to accommodate ancestor tables.
-          // 1em + 10px is an eyeball measurement at font sizes 14 and 18
-          // (Maybe the 10px is from .content padding-left?)
-          width: isTableCol1 ? width : `calc(100% - ${x}px + 1em + 10px)`,
-          ...style,
-          textAlign: isTableCol1 ? 'right' : undefined,
-        }}
+      <ThoughtNode
+        isLastActionSwapParent={isLastActionSwapParent}
+        isLastActionNewThought={isLastActionNewThought}
+        isParentSwapped={isParentSwapped}
+        isLastActionEditThought={isLastActionEditThought}
+        isLastActionSetCursor={isLastActionSetCursor}
+        x={x}
+        y={y}
+        width={width}
+        isTableCol1={isTableCol1}
+        style={style}
+        fadeThoughtRef={fadeThoughtRef}
+        dragInProgress={dragInProgress}
+        autofocusDepth={autofocusDepth}
+        depth={depth}
+        cliff={cliff}
+        path={path}
+        isTableCol2={isTableCol2}
+        isLastVisible={isLastVisible}
+        cursorIsGreaterThanPrevSibling={cursorIsGreaterThanPrevSibling}
+        treeThoughtsPositioned={treeThoughtsPositioned}
+        index={index}
       >
-        <div ref={fadeThoughtRef}>
-          <VirtualThought
-            debugIndex={testFlags.simulateDrop ? indexChild : undefined}
-            depth={depth}
-            dropUncle={thoughtId === cursorUncleId}
-            env={env}
-            indexDescendant={indexDescendant}
-            // isMultiColumnTable={isMultiColumnTable}
-            isMultiColumnTable={false}
-            leaf={leaf}
-            onResize={setSize}
-            path={path}
-            prevChildId={prevChild?.id}
-            showContexts={showContexts}
-            simplePath={simplePath}
-            singleLineHeight={singleLineHeightWithCliff}
-            // Add a bit of space after a cliff to give nested lists some breathing room.
-            // Do this as padding instead of y, otherwise there will be a gap between drop targets.
-            // In Table View, we need to set the cliff padding on col1 so it matches col2 padding, otherwise there will be a gap during drag-and-drop.
```

Comment:
Was it intentional to remove some of these comments? Unless I'm missing something, this seems like an example of one which is still relevant.

Reply:
This was unintentional. I've added back comments.